swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -291,14 +291,14 @@ extension Adapter {
     } else {
       Log.tunnel.log("\(#function): Detected network change: Online.")
 
-      // Hint to connlib we're back online, but only do so if our primary interface changes,
-      // and therefore we need to bump sockets. On darwin, this is needed to send packets
+      // Hint to connlib we're back online, but only do so if our connectivity has
```

Comment:
The "hint" wording here is kind of outdated as "reset" is not conditional on anything, we just clear all network state.